perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/model/PerunAttribute.java

```diff
@@ -74,6 +74,7 @@ public interface PerunAttributeNames {
 		public static final String ldapAttrAdminOfGroup = "adminOfGroup";
 		public static final String ldapAttrAdminOfFacility = "adminOfFacility";
 		public static final String ldapAttrUuid = "uuid";
+		public static final String ldapAttrUserEligibilities = "userEligibilities";
```

Comment:
Here you should be able to use the variable _perunAttrUserEligibilities_ defined on line 27 instead of defining the same string again. Should be a non-issue, but would be more consistent with other Perun attribute names also used in this way in LDAP attribute names definitions.